src/js/06-code.js

```diff
@@ -278,4 +303,115 @@ document.addEventListener('DOMContentLoaded', function () {
 
       elements[0].classList.add('tabbed-target--active')
     })
+
+  //
+  // Tabbed sections in the drivers manual
+  //
+
+  if (storageAvailable('sessionStorage')) {
+    var sessionStorage = window.sessionStorage
+  }
```

Comment:
I think this block could be removed and we should use `window.sessionStorage.getItem('code_example_language')` below